lib/src/presentation/view/genre_movie_list.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+
+import '../../core/parameter/discover_parameters.dart';
+import '../../core/util/enums.dart';
+import '../../core/util/strings.dart' as strings;
+import '../../domain/entity/event/implementation/movie_event.dart';
+import '../../domain/entity/genre.dart';
+import '../../domain/entity/movie.dart';
+import '../bloc/movies_by_genre_bloc.dart';
+import '../widget/detailed_list_item.dart';
+
+class GenreList extends StatefulWidget {
+  final Genre genre;
+  static const double listHorizontalPadding = 8;
+
+  const GenreList({
+    super.key,
+    required this.genre,
+  });
+
+  @override
+  State<GenreList> createState() => _GenreListState();
+}
+
+class _GenreListState extends State<GenreList> {
+  final MoviesByGenreBloc bloc = Get.find();
+
+  @override
+  void initState() {
+    super.initState();
+    bloc.initialize();
```

Comment:
you can made the initialization at the same time that you create all the instances